models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb

```diff
@@ -50,16 +50,16 @@
    "source": [
```

Comment:
<p>Make sure these are sorted the same way they would be by <code>isort</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nv-morpheus/Morpheus/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>